app/_includes/md/kic/add-certificate.md

```diff
@@ -3,51 +3,58 @@ over HTTPS. This is not required, as {{site.base_gateway}} will serve a default
 certificate if it cannot find another, but including TLS configuration along
 with routing configuration is typical.
 
-First, create a test certificate for the `{{ include.hostname }}` hostname using one of the following commands:
+1. Create a test certificate for the `{{ include.hostname }}` hostname.
 
+ {% capture the_code %}
 {% navtabs codeblock %}
 {% navtab OpenSSL 1.1.1 %}
 ```bash
-openssl req -subj '/CN={{ include.hostname }}' -new -newkey rsa:2048 -sha256 \
```

Comment:
@Rajakavitha1 Why have we switched from `include.hostname` to hardcoded values here?

Reply:
That was a copy paste error. Thanks for pointing this.